Sources/AudioKit/Nodes/Mixing/Mixer.swift

```diff
@@ -33,17 +37,18 @@ public class Mixer: Node, Toggleable {
     }
 
     /// Initialize the mixer node with no inputs, to be connected later
-    public init(volume: AUValue = 1.0) {
+    public init(volume: AUValue = 1.0, name: String? = nil) {
```

Comment:
I would recommend making this non-optional, but provide the default for the node. "Mixer" is a good choice that follows the existing conventions.